src/pages/dashboard/Explore.tsx

```diff
@@ -49,46 +67,70 @@ const StyledList = styled(List)`
 
 const Explore = () => {
   const firstPage = 1;
```

Comment:
en liten btw: når konstanter ligger her vil de opprettes på nytt for hver render, så hvis det aldri skal endres er det ofte like greit å legge den utenfor komponenten. en variabel er ikke kostbare greiene da, men en større funksjon kan fort påvirke performance mer